ww3tools/ProcSat_Altimeter.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+ProcSat_Altimeter.py
+
+VERSION AND LAST UPDATE:
+ v1.0  04/04/2022
+ v1.1  07/18/2022
+ v1.2  05/21/2023
+ v1.3  10/11/2023
+ v2.0  10/13/2023
+
+PURPOSE:
+ Script to process altimeter tracks. Two options are available:
+ (1) It can collocate tracks into a regular lat/lon grid (a ww3 sample output
+   or gridInfo.nc, generated by preprGridMask.py, must be entered)
+ (2) Or keep the observation space as reference and produce averages
+   along the track.
+ This script uses confirguration inputs saved in ww3tools.yaml
+  and the reading module wread.py
+ A total of 15 satellite missions are listed below. The period of each
+  altimeter can be verified at:
+  https://www.sciencedirect.com/science/article/pii/S0273117721000594
+  https://ars.els-cdn.com/content/image/1-s2.0-S0273117721000594-gr1_lrg.jpg
+
+USAGE:
+ This program processes one satellite mission per run, entered as 
+  argument (only the ID, sys.argv), Select only one:
+  JASON3,JASON2,CRYOSAT2,JASON1,HY2,SARAL,SENTINEL3A,ENVISAT,ERS1,ERS2,GEOSAT,GFO,TOPEX,SENTINEL3B,CFOSAT
+ Altimeters must have been previously downloaded (see wfetchsatellite_AODN_Altimeter.sh)
+ User must inform if wants to target the model space (tspace=1) or the satellite
+  space (tspace=2) in ww3tools.yaml.
+ Quality control can be selected or excluded in ww3tools.yaml, and user
+  can modify satellite quality control parameters in ww3tools.yaml as well.
+ The path where AODN altimeter data is located must be informed in ww3tools.yaml
+ The output path where the results are saved must be informed in ww3tools.yaml
+ INPUT ARGUMENTS are (the first one is mandatory):
+ (1) satellite name
+ (2) initial date ('YYYYMMDDHH')
+ (3) final date ('YYYYMMDDHH')
+ (4) time-step for the time array of results
+ (5) ww3 grid output sample file or gridInfo.nc
+
+ The ww3tools.yaml file must be saved (or symbolic link) in the same dir.
```

Comment:
Is there a reason that the location and name of this yaml file is not an input?   Is there a reason there are other inputs beyond just simply this file name?

Reply:
The initial idea was to follow the same structure as WW3. So the python scripts will read a .inp (in this case, a .yaml) that must be saved (or symbolic link) in the same directory. This is explained in the header. If it is still not clear I can quickly add an option of informing the location/name of ww3tools.yaml , as an extra input argument. @Ghazal-Mohammadpour, @JessicaMeixner-NOAA, and @sbanihash let me know what you prefer.

Reply:
I guess I'd counter to say that WW3 doesn't always have the current latest and greatest or best practices always, but I see what you are saying here.  If backwards compatibility or changing up structure is not an issue, we can always leave this for something to think over and change later.

Reply:
@ricampos I'm going to resolve this conversation as something we can change in the future -- as long as you are okay if things change and are not backwards compatible.  If that's an issue we should discuss now before this is merged.